ledger/src/blockstore.rs

```diff
@@ -156,6 +156,24 @@ impl PossibleDuplicateShred {
     }
 }
 
+enum WorkingStatus<T> {
+    Dirty(T),
+    Clean(T),
+}
+
+impl<T> WorkingStatus<T> {
+    fn unwrap(&self) -> &T {
+        match self {
+            Self::Dirty(value) => value,
+            Self::Clean(value) => value,
+        }
+    }
```

Comment:
This seems a confusing overload of `unwrap`.
Can we instead implement `From`?
https://doc.rust-lang.org/std/convert/trait.From.html
something like:
```rust
impl<T> From<WorkingStatus<T>> for T { ...
```

Reply:
I can't implement `From` as it violates the orphan rule. I've changed the names a bit to `WorkingEntry` and `fn value` hopefully that's less confusing.